docs/8.4-breaking-changes.md

```diff
@@ -0,0 +1,150 @@
+# Breaking and incompatible changes in {{vers}}
+
+Review these items before upgrading from 8.0 to {{vers}}. This document covers breaking behavioral changes, removed features, and removed variables that may affect your upgrade. Each entry includes the impact, replacement (if available), and recommended action.
```

Comment:
For me it is not clear from reading this whether we cover ALL breaking changes, or only MAJOR ones.
IMO it is better to clarify this, e.g. by saying, "... covers most significant breaking ..." and also reference the Upstream manual for the complete list.